server/routes/cbauthentication/cb_password_change.js

```diff
@@ -13,7 +13,7 @@ const schemas = {
     token: Joi.string().required(),
 
     password: Joi.string()
-      .regex(/^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&*])(?=.{8,})/, 'strong_pwd')
+      .regex(/^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!#$%&()*+,-./:;<=>?@[\]^_{|}~?])(?=.{8,})/, 'strong_pwd')
```

Comment:
Do `$`, `^`, `?`, `.`, `*`, need to be escaped? Also you've `?` twice.

Reply:
@eliascodes quantifiers don't work in character sets, but I have missed a few escapes.